admin/catalog.php

```diff
@@ -125,12 +125,12 @@
             foreach ($songs as $song_id) {
                 Song::update_enabled(true, $song_id);
             }
-            $body = count($songs) . nT_(' Song Enabled', ' Songs Enabled', count($songs));
+            $body = count($songs) . ' ' . nT_('Song Enabled', 'Songs Enabled', count($songs));
         } else {
-            $body = T_('No Disabled Songs selected');
+            $body = T_('No disabled songs selected');
         }
         $url      = AmpConfig::get('web_path') . '/admin/catalog.php';
-        $title    = count($songs) . nT_(' Disabled Song Processed', ' Disabled Songs Processed', count($songs));
+        $title    = count($songs) . ' ' . nT_('Disabled Song Processed', 'Disabled Songs Processed', count($songs));
```

Comment:
"Processed" should probably be lowercase

Reply:
agree